src/pages/progress-connect/index.css

```diff
@@ -1,3 +1,10 @@
+@font-face {
+  font-family: 'Titillium Web';
+  src: url('../../assets/fonts/titillium-web/titillium-web-latin-400-normal.woff2')
+    format('woff2');
+  font-style: normal;
+  font-weight: 400;
+}
```

Comment:
Is this not working if we add this below the `@tailwind` parts?

Reply:
sure. no problemo 👍